bounties/0018 Ledger Metrics.md

```diff
@@ -1,8 +1,12 @@
-# Ledger Metrics
-
-**Category:** Technical Documentation
+---
+ID: "0018"
+Discussion: https://github.com/XRPLBounties/Proposals/discussions/0018
+Category: Technical Documentation
```

Comment:
Should this fall into 'Technical Documentation' or 'New Feature', since it adds new fields to the Explorer front-end?

Reply:
New Feature sounds good - thanks!